backend/db/data/books/1.json

```diff
@@ -10,71 +10,71 @@
     },
     {
       "title": "Norbert the Norwegian Ridgeback",
-      "order": "14"
+      "summary": "Hagrid wins a dragon egg in a card game and becomes obsessed with hatching it. As the egg is about to hatch, Hagrid asks Harry and Hermione to help him smuggle the dragon, named Norbert, out of the castle to Charlie Weasley, who works with dragons in Romania. The trio's late-night excursion to the tower where Norbert is kept leads to a dangerous encounter with Draco Malfoy. They successfully pass Norbert to Charlie's friends, but they get caught by Mr Filch."
     },
     {
       "title": "The Forbidden Forest",
-      "order": "15"
+      "summary": "Harry and Hermione serve detention in the Forbidden Forest as punishment for being out of their common room after hours. They are accompanied by Hagrid and encounter dangerous creatures, including a wounded unicorn and a mysterious centaur named Firenze. During their time in the forest, they witness a hooded figure drinking the blood of the unicorn. Firenze explains that drinking unicorn blood allows a person to prolong their life, but at a terrible cost - the drinker becomes cursed. Harry begins to suspect that Snape wants to obtain the Philosopher's Stone to help Voldemort regain his strength."
     },
     {
       "title": "Through the Trapdoor",
-      "order": "16"
+      "summary": "Harry, Ron, and Hermione, armed with the knowledge of Fluffy guarding the trapdoor, embark on a perilous journey to reach the Philosopher's Stone before it falls into the wrong hands. These include Devil's Snare, a plant that tries to strangle them, a life-sized wizard's chess match, a game of flying keys, and potions that must be consumed in the correct order to proceed. In the last puzzle it turns out that only Harry can proceed further, and Hermione decides to turn back to Ron. Harry encounters an unexpected figure waiting for him."
     },
     {
       "title": "The Man with Two Faces",
-      "order": "17"
+      "summary": "Harry confronts Professor Quirrell, who is attempting to steal the Stone. Harry faces Voldemort, who is attached to the back of Quirrell's head. Voldemort demands that Harry give him the Philosopher's Stone, but Harry's courage and selflessness protect the Stone from falling into Voldemort's grasp. The Philosopher's Stone is destroyed to prevent it from falling into the wrong hands. Dumbledore explains to Harry that his mother's love and the protection it provided played a crucial role in safeguarding the Stone."
```

Comment:
here it should also reveal that Quirrell actually is "having Voldemort on the back of his head", you know what I mean?

Reply:
@danielschuster-muc actually I have in the summary "Harry faces Voldemort, who is attached to the back of Quirrell's head." but maybe I can rework this sentence

Reply:
Oops, overread it, sorry